extensions/wikia/ContentReview/tests/ContentReviewHelperTest.php

```diff
@@ -98,96 +156,113 @@ public function testHasPageApprovedId( $latestReviewedId, $oldId, $expected, $me
 		$this->assertEquals( $expected, $this->getHelper()->hasPageApprovedId( $modelMock, 0, 0, $oldId ), $message );
 	}
 
+	public function shouldPageContentBeReplacedData() {
+		$goodMimeType = 'text/javascript';
+		$badMimeType = 'text/css';
+		return [
+			[
+				false, // isJsPage()
+				false, // inNamespace()
+				$badMimeType,
+				false, // expected
+			],
+			[
+				false, // isJsPage()
+				true, // inNamespace()
+				$badMimeType,
+				false, // expected
+			],
+			[
+				false, // isJsPage()
+				false, // inNamespace()
+				$goodMimeType,
+				false, // expected
+			],
+			[
+				false, // isJsPage()
+				true, // inNamespace()
+				$goodMimeType,
+				true, // expected
+			],
+			[
+				true, // isJsPage()
+				false, // inNamespace()
+				$badMimeType,
+				true, // expected
+			],
+		];
+	}
+
 	public function replaceWithLastApprovedRevisionProvider() {
-		$pageId = 123;
-		$revId1 = 566;
-		$revId2 = 567;
-		$revIdNull = null;
-		$textEmpty = '';
-		$text1 = '';
-		$text2 = '';
-		$jsType = 'jstype';
+		$originalText = 'This is the original text.';
+		$lastReviewedText = 'This is the last reviewed text.';
+		$emptyText = '';
+
 		return [
+			/**
+			 * Case 1 - a page does not qualify to be reviewed (isPageReviewed === false).
```

Comment:
I'd pass these comments to the test method and using them with the assert as they provide context when the test case fails when running the tests (either manually or in Jenkins).